cypress/integration/seededFlows/adminFlows/users/manageCredits.spec.js

```diff
@@ -15,7 +15,7 @@ describe('Manage User Credits', () => {
       cy.fixture('users/adminUser.json').as('user');
       cy.get('@user').then((user) => {
         cy.loginAndVisit(user, '/admin/users');
-        cy.findByRole('link', { name: 'credits_user' }).click();
+        cy.findAllByText('Credits User').first().click();
```

Comment:
@aitchiss I was expecting this to work with `cy.findByRole('link', { name: 'Credits User' }).first().click();` with our updated changes but I couldn't get it to work that way 😞 It works with the text but I assume that this is not the optimal way to do it. Is there any obvious reason you can see why it wouldn't work with the role?

Reply:
Did you try `cy.findAllByRole('link', { name: 'Credits User' }).first().click();` (with the 'all')? What does the error output say 🤔

Reply:
🤦🏽‍♀️